config/config.exs

```diff
@@ -31,6 +31,27 @@ config :animina, :ai_message_help_price, 20
 # Configures whether to have ai automatically respond to messages
 config :animina, :autoreply_messages_with_ai, false
 
+# Configures default maximum age for potential partners
+config :animina, :default_potential_partner_maximum_age, 100
+
+# Configures default minimum age for potential partners
+config :animina, :default_potential_partner_minimum_age, 18
+
+# confugures the difference in age if no maximum or minimum age is set
+config :animina, :default_potential_partner_age_difference, 10
+
+# Configures the maximum height for potential partners
+config :animina, :default_potential_partner_maximum_height, 300
+
+# Configures the minimum height for potential partners
+config :animina, :default_potential_partner_minimum_height, 0
```

Comment:
Doesn't make sense either. Plus what is this variable about? Is this the default value for the form or for some sort of validation?

Reply:
No , this value is the one we use when the page loads or the user has not set any height , we want to have the lowest number possible as a minimum height and the highest number as the maximum height so by default we show as many users as possible